src/js/jsx/sections/export/ExportAllPanel.jsx

```diff
@@ -202,6 +205,7 @@ define(function (require, exports, module) {
          */
         _setUseArtboardPrefix: function (event, enabled) {
             this.getFlux().actions.export.setUseArtboardPrefix(enabled);
+            headlights.logEvent("export", "batch-export", "prefix-enabled-" + enabled);
```

Comment:
I suspect these kinds of events will not be super useful... What are we going to count, the number of times someone toggled this setting? But... I guess I don't really have a better short-term suggestion. So it goes.